src/js/components/List/stories/onClickItem.js

```diff
@@ -25,14 +25,28 @@ for (let i = 0; i < 40; i += 1) {
 
 export const OnClickItemList = () => {
   const [clicked, setClicked] = React.useState();
+  const [show, setShow] = React.useState();
+
   return (
     <Grommet theme={grommet}>
       <Box align="center" pad="large" gap="large">
         <List
           data={data.slice(0, 10)}
-          onClickItem={event => setClicked(event.item)}
+          onClickItem={event => {
+            setShow(true);
+            console.log('=== Modal Opened ===');
+            setClicked(event.item);
+          }}
         />
 
+        {show && (
+          <Layer
+            onEsc={() => setShow(false)}
+            onClickOutside={() => setShow(false)}
+          >
```

Comment:
Can we add something inside the Layer indicating what the active object is? I think that will reinforce what's going on.

Reply:
To clarify, would you like to display what the current active item was before the Layer opened inside the Layer?

Reply:
No, I was wanting to only show the detail of the item inside the Layer. I think we can remove the previous behavior of showing it below the List.

Reply:
Ahh gotcha! Will do